cypress/integration/learn/challenges/sass.js

```diff
@@ -0,0 +1,28 @@
+const location =
+  '/learn/front-end-development-libraries/sass/' +
+  'use-for-to-create-a-sass-loop';
+
+const getIframeBody = () => {
+  return cy
+    .get('.challenge-preview iframe')
+    .its('0.contentDocument')
+    .should('exist')
+    .its('body')
+    .should('not.be.undefined')
+    .then(cy.wrap);
+};
+
+describe('Sass Challenge', () => {
+  before(() => {
+    cy.visit(location);
+    cy.wait(5000);
+  });
+
+  it('should render the sass preview', () => {
+    getIframeBody().find('.text-1');
+    getIframeBody().find('.text-2');
+    getIframeBody().find('.text-3');
+    getIframeBody().find('.text-4');
+    getIframeBody().find('.text-5');
+  });
```

Comment:
```suggestion
  it('should render the sass preview', () => {
    const iframeBody = getIframeBody();
    iframeBody.find('.text-1');
    iframeBody.find('.text-2');
    iframeBody.find('.text-3');
    iframeBody.find('.text-4');
    iframeBody.find('.text-5');
  });
```

Reply:
I had tried that, but it didn't work because the first `find` updated the reference of that `iframeBody` variable to point to the `p` element it found.

Essentially it's chaining `.find()` calls.